fix-appdata.patch

```diff
@@ -0,0 +1,43 @@
+diff -ruN a/share/appdata/scilab.appdata.xml b/share/appdata/scilab.appdata.xml
+--- a/share/appdata/scilab.appdata.xml	2021-07-15 19:48:32.000000000 +0700
++++ b/share/appdata/scilab.appdata.xml	2021-07-23 00:21:55.239705239 +0700
+@@ -1,7 +1,7 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <!-- Copyright 2013 Clement DAVID <clement.david@scilab-enterprises.com> -->
+ <component type="desktop-application">
+-    <id>org.scilab.www</id>
++    <id>org.scilab.Scilab</id>
```

Comment:
did you submit this patch upstream? It would be good to be consistent with upstream on the app-id in the future.

Reply:
have not, I think they use mailing list, I'm not familiar with it, but I will submit a request.